### DISCUSSION TITLE
Prevent blocked tumblrs from ever appearing in the timeline in any way

### DISCUSSION THREAD

**Discussion by RagingSalamence** [AprilSylph/XKit-Rewritten#1103] [Suggestion Box]
Hi, I'd like a feature on Xkit that would prevent a blocked blog from showing up in the timeline and in any blog, be it because someone else reblogged a post from them, answered an ask from them, or was merely tagged on it. If I block a blog I'd rather not have them show up in my timeline by coincidence.

**Comment by chungusg** [comment]
I also would love to see this! I mostly block blogs because I'm in fandom where dni's for ships are super common, and I never want to see or interact with those blogs...ever! Even through a "safe" mutual's reblogs! 

For now, I suggest filtering url's in Tumblr's native content filter option, but this is still obtrusive (since you can still see that the content was filtered) and simply INEFFECTIVE if you block lots of blogs (like me...)
